community-containers/dlna/readme.md

```diff
@@ -1,5 +1,11 @@
 ## DLNA server
-This container bundles DLNA server for your Nextcloud files to be accessible by the clients in your local network. Simply run the container and look for a new media server Nextcloud-AIO in your local network.
+This container bundles DLNA server for your Nextcloud files to be accessible by the clients in your local network. Simply run the container and look for a new media server `nextcloud-aio` in your local network.
+
+### Notes
+- This container needs the `nextcloud-aio-nextcloud` container to be up and running to work
```

Comment:
I would remove this notice. As I said in best case there should be a check maybe in the startup command of the container that checks if for example the given nextcloud domain reachable and only start up the service in that case?

Reply:
You could use netcat-openbsd like this for example:
```bash
while ! nc -z "$NC_DOMAIN" 443; do
	echo "Waiting for Nextcloud to become online"
	sleep 5
done
```

Reply:
In that case you would need to add a wrapper bash script as entrypoint and start the dlna server from there.

Reply:
If a domain is given, it could run this check and wait for it to start. If not, it should skip this waiting...

Reply:
But I would have to put this script into the nextcloud-dlna container, right? I would rather not do that in the nextcloud-dlna container for two reasons:
* nextcloud-dlna doesn't need an access to the Nextcloud APP via HTTP, so checking it and waiting for it in most cases would make no sense (it only needs an access to the DB and `data` folder)
* Nextcloud is not always exposed on 443 port, there is a chance that it's a backend installation using port 80 (or other).

Reply:
I've created a PR: https://github.com/thanek/nextcloud-dlna/pull/17. Let me know what you think :)

Reply:
Nice one! I'll take a look at it and let you know, looks pretty good at first glance :)

Reply:
Great. I've just released version 1.1.2 of nextcloud-dlna - it contains your wrapper and is aware of nextcloud presentce. Thanks!

Reply:
nice :)